components/elm/src/main/elm_driver.F90

```diff
@@ -199,6 +217,13 @@ subroutine elm_drv(doalb, nextsw_cday, declinp1, declin, rstwr, nlend, rdate)
     ! the calling tree is given in the description of this module.
     !
     ! !USES:
+#if defined(CLDERA_PROFILING)
+    use SurfaceRadiationMod, only: surfrad_type
+    use SolarAbsorbedType ,  only : solarabs_type
+    use VegetationDataType,  only: vegetation_energy_state, vegetation_water_flux
+    use PhotosynthesisType,  only: photosyns_type
+    use CanopyStateType,     only: CanopyState_type
+#endif
```

Comment:
are we using these? and some of the cldera funcs above?

Reply:
nope, I forgot I left these in here after I moved things over to the elm init